2017/neural-networks/spanish/auto_generated.srt

```diff
@@ -19,1158 +19,1154 @@ Y quiero que te tomes un momento para apreciar lo loco
+00:00:42,890 --> 00:00:48,260
+misma idea, al tiempo que reconoce otras imágenes como ideas propias y distintas.
 
 12
-00:00:44,642 --> 00:00:48,260
-tiempo reconoce otras imágenes como sus propias ideas distintas.
+00:00:49,220 --> 00:00:54,851
+Pero si te dijera, oye, siéntate y escribe para mí un programa que tome una cuadrícula 
 
 13
-00:00:49,220 --> 00:00:53,337
-Pero si te dijera, oye, siéntate y escribe para mí un programa que 
+00:00:54,851 --> 00:00:59,059
+de 28x28 píxeles como ésta y emita un único número entre 0 y 10, 
 
 14
-00:00:53,337 --> 00:00:58,621
-tome una cuadrícula de 28x28 píxeles como esta y genere un único número entre 0 y 10, 
+00:00:59,059 --> 00:01:04,432
+diciéndote cuál cree que es el dígito, pues la tarea pasa de cómicamente trivial a 
 
 15
-00:00:58,621 --> 00:01:02,923
-diciéndote cuál cree que es el dígito, bueno, la tarea va de desde lo 
+00:01:04,432 --> 00:01:06,180
+desalentadoramente difícil.
 
 16
-00:01:02,923 --> 00:01:06,180
-cómicamente trivial hasta lo abrumadoramente difícil.
+00:01:07,160 --> 00:01:09,099
+A menos que hayas estado viviendo bajo una roca, 
 
 17
-00:01:07,160 --> 00:01:09,109
-A menos que hayas estado viviendo bajo una roca, 
+00:01:09,099 --> 00:01:11,513
+creo que apenas necesito motivar la relevancia e importancia 
 
 18
-00:01:09,109 --> 00:01:11,496
-creo que no necesito motivar la relevancia y la importancia 
+00:01:11,513 --> 00:01:14,640
+del aprendizaje automático y las redes neuronales para el presente y el futuro.
 
 19
-00:01:11,496 --> 00:01:14,640
-del aprendizaje automático y las redes neuronales para el presente y el futuro.
+00:01:15,120 --> 00:01:18,467
+Pero lo que quiero hacer aquí es mostrarte lo que es realmente una red neuronal, 
 
 20
-00:01:15,120 --> 00:01:18,616
-Pero lo que quiero hacer aquí es mostrarles qué es realmente una red neuronal, 
+00:01:18,467 --> 00:01:21,980
+suponiendo que no tengas conocimientos previos, y ayudarte a visualizar lo que hace, 
 
 21
-00:01:18,616 --> 00:01:21,848
-sin asumir ningún trasfondo, y ayudar a visualizar lo que está haciendo, 
+00:01:21,980 --> 00:01:24,460
+no como una palabra de moda, sino como una pieza matemática.
 
 22
-00:01:21,848 --> 00:01:24,460
-no como una palabra de moda sino como una pieza matemática.
+00:01:25,020 --> 00:01:28,608
+Mi esperanza es que salgas sintiendo que la propia estructura está motivada, 
 
 23
-00:01:25,020 --> 00:01:28,126
-Mi esperanza es que usted salga sintiendo que la estructura en sí 
+00:01:28,608 --> 00:01:31,590
+y que sientas que sabes lo que significa cuando lees o escuchas 
 
 24
-00:01:28,126 --> 00:01:31,233
-está motivada y que sienta que sabe lo que significa cuando lee o 
+00:01:31,590 --> 00:01:34,340
+hablar de una red neuronal, entre comillas, de aprendizaje.
 
 25
-00:01:31,233 --> 00:01:34,340
-escucha acerca del aprendizaje entre comillas de una red neuronal.
+00:01:35,360 --> 00:01:38,422
+Este vídeo sólo va a estar dedicado al componente de estructura, 
```

Comment:
This, again, yells "Spaniard": "vídeo" is used pretty much only in Spain ([reference - in Spanish](https://www.rae.es/dpd/video)). Following the logic of folks contributing to Spanish translations, "video" should be preferred as: 1. There are more viewers from Latin America than Spain, 2. There are far more Spanish speakers in The Americas than in Spain.

```suggestion
Este video sólo va a estar dedicado al componente de estructura, 
```